okdownload/src/main/java/com/liulishuo/okdownload/core/Util.java

```diff
@@ -436,7 +436,12 @@ public static void addUserRequestHeaderField(@NonNull Map<String, List<String>>
                                                  @NonNull DownloadConnection connection)
             throws IOException {
         inspectUserHeader(userHeaderField);
+        addUserRequestHeaderFieldWithoutInspect(userHeaderField, connection);
```

Comment:
I don't think this isn't always user-request fields, use this name would be confused in the future.

Reply:
updated